src/components/Navigation/Navigation.js

```diff
@@ -276,7 +276,7 @@ export const Navigation = ({
         room,
       }}
     />
-    {inventoryLimit > -1 && (
+    {inventoryLimit > INFINITE_STORAGE_LIMIT && (
```

Comment:
As a note to myself, this logic looks a little weird given the `-1` is now hidden behind a named constant (as it should be). I'll probably make this a `!==` comparison later, but for this PR I think it's better to avoid any logic changes.